package.json

```diff
@@ -53,7 +53,7 @@
   },
   "scripts": {
     "analyze": "NODE_ENV=production NETWORK=main ANALYZE=1 node --max_old_space_size=4096 node_modules/.bin/webpack --config webpack.prod.config.js --progress --colors",
-    "build": "webpack --config webpack.prod.config.js",
+    "build": "node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js  --config webpack.prod.config.js",
```

Comment:
@jelle changed this because builds on Heroku were breaking with this....

Reply:
I had multiple reasons to believe it would fix the problem I'm seeing with review apps.  but I'm not sure it has.  waiting to get permissions on heroku to see what is going on.

Reply:
But note that this *does* fix the ability to build for prod locally.

Reply:
It is not so unusual for me to want to run a local prod build